base/dsp.jl

```diff
@@ -121,9 +121,10 @@ function deconv{T}(b::StridedVector{T}, a::StridedVector{T})
 end
 
 """
-    conv(u,v)
+    conv(u, v)
 
-Convolution of two vectors. Uses FFT algorithm.
+Convolves two vectors. (If `u`, `v` are the coefficients of two polynomials, `conv(u, v)`
```

Comment:
Perhaps "(If `u` and `v` contain the coefficients of two polynomials, `conv(u, v)` yields the coefficients of the product of those polynomials.)"?